js/webpack.prebuilt.config.js

```diff
@@ -11,6 +11,17 @@ const rules = [
           importLoaders: 1,
         },
       },
+      {
+        // Prefix antd's global styles to keep them from interfering with notebook styling
+        loader: "prefix-css-loader",
+        options: {
+          // selectors inside :where do not affect specificity - i.e. prefixed rules will not be reordered in priority.
+          // NOTE: :where was broadly supported by browsers only after ~Jan 2021
+          selector: ":where(.cell-viewer)",
+          exclude: /[.#]/, // Only prefix selectors that refer only to tags
+          minify: false,
```

Comment:
i wonder if we can do this only in one of the two webpack configs.  ideally it goes in the non-prebuilt one, but if we can get rid of one it would be great.

Reply:
could go in a shared config file. Importing from one to the other could get hacky.

Reply:
e.g. it's possible to have only a single source of truth for less loading rules by replacing the entire `rules` definition in webpack.prebuilt.config.js with the line `const rules = require("./webpack.config")[0].module.rules.filter(({ test }) => test.source === "\\.less$");`. This is satisfyingly brief, but is it too arcane to be worthwhile?

Reply:
I didn't realize that it actually HAS to be in both files.  if it has to be in both, then I guess I prefer it as-is.